client.go

```diff
@@ -298,10 +314,28 @@ func (client *client) NetworkParams() *chaincfg.Params {
 func (client *client) FormatTransactionView(msg, txhash string) string {
 	switch client.NetworkParams().Name {
 	case "mainnet":
-		return fmt.Sprintf("%s, https://www.blockchain.com/btc/tx/%s", msg, txhash)
+		return fmt.Sprintf("%s, transaction can be viewed at https://www.blockchain.com/btc/tx/%s", msg, txhash)
 	case "testnet3":
 		return fmt.Sprintf("%s, transaction can be viewed at https://testnet.blockchain.info/tx/%s", msg, txhash)
 	default:
-		panic(fmt.Sprintf("Unsupported network: %s", client.NetworkParams().Name))
+		panic(NewErrUnsupportedNetwork(client.NetworkParams().Name))
+	}
+}
+
+func (client *client) exponentialBackoff(ctx context.Context, f func() (interface{}, error)) (interface{}, error) {
```

Comment:
I would favour calling this function `backoff` instead of `exponentialBackoff`. The prefix `exponential` leaks the implementation details of the function (in future, what if we change this to be linear?).

Reply:
done